ecl/hql/hqllex.l

```diff
@@ -1929,18 +1929,13 @@ FUNCTIONMACRO|MACRO {
 
                         if (likely(lookupFlags & LEXstring))
                         {
+                            //Could common up, because except for this warning this code is identical to the token above...
                             lexer->reportWarning(CategorySyntax, returnToken, ERR_HEXDATA_ODDDIGITS,"hex data must have even number of hex digits");
```

Comment:
Should the error message be "had data **should** have an even number" if it is just a warning?

Reply:
Fixed.